### ISSUE TITLE
Multiple Form Labels - Create New Generic Work

### ISSUE THREAD

**Issue by mtribone** [samvera-labs/nurax-pre2023#125]
### Descriptive summary

When creating a new generic work and using the WAVE accessibility toolbar, there are several form controls that have _multiple_ form labels.

They are:

1. Embargo
1. Lease

### Rationale

A form control should have at most one associated label element. If more than one label element is associated to the control, assistive technology may not read the appropriate label.

### Expected behavior

All form controls should have one label with content for each form control.

### Actual behavior
A form control has more than one label associated with it.

### Steps to reproduce the behavior

Select Works from the left-hand menu
On the works screen select "Add new work"
Select "generic Work" from the modal
Validate the form with the WAVE toolbar

### Related work
#123, #124
